src/erlfmt_cli.erl

```diff
@@ -58,7 +61,23 @@ do(Name, Opts) ->
 do(Name, PreferOpts, DefaultOpts) ->
     PreferParsed = parse_opts(PreferOpts),
     DefaultParsed = parse_opts(DefaultOpts),
-    Parsed = resolve_parsed(PreferParsed, DefaultParsed),
+    case {specified_files(PreferOpts), PreferParsed} of
+        {SpecifiedFiles, {format, [], _}} when SpecifiedFiles =/= [] ->
```

Comment:
If `PreferParsed` returns an empty list of files to format, do we need to run the `specified_files` check? What's the case when they would return different values?

Reply:
If you do not specify files, we are going to add the default files, but we do not want to add these default files in cases where you specified files that just didn't match any existing files.